### ISSUE TITLE
Documentation for Model.trainable_weights is missing

### ISSUE THREAD

**Issue by worldpeaceaspirer** [tensorflow/tensorflow#49890]
## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/api_docs/python/tf/keras/Model

## Description of issue (what needs changing): 
**`model.trainable_weights`** returns the **`Weights`** and **`Biases`** of all the **`Layers`** of the **`Model`**. Documentation to that function is missing in [Tensorflow.org](https://www.tensorflow.org/api_docs/python/tf/keras/Model).

**Comment by tusharvickey1999** [comment]
@worldpeaceaspirer `trainable_weights` is the property of `tf.keras.layers.Layer` class, which is the super class of `tf.keras.models` and that's why you can call it using `model.trainable_weights`. 
The property is listed in the documentation's attribute sections at this link : [tensorflow.org](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer) 

hope this helps 😀.

**Comment by UsharaniPagadala** [comment]
@worldpeaceaspirer 

Could you please check the above comment and let us know if it helps.Thanks

**Comment by worldpeaceaspirer** [comment]
@tusharvickey1999,
Thank you for your response but still the **documentation** on what exactly **`model.trainable_weights`** or **`layer.trainable_weights`** does and what does that **`function`** **`Return`**, is missing.

**Comment by tusharvickey1999** [comment]
@worldpeaceaspirer it's is stated under the attribute that it gets you list of all the trainable_weights/variables for the layer/model.

**Comment by ymodak** [comment]
@worldpeaceaspirer You may want to refer usage examples listed on the same page https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer#examples
Example 1 and 3 talks about implementing  `trainable_weights` and `non_trainable_weights`

**Comment by worldpeaceaspirer** [comment]
> @tusharvickey1999,
> Thank you for your response but still the **documentation** on what exactly **`model.trainable_weights`** or **`layer.trainable_weights`** does and what does that **`function`** **`Return`**, is missing.

I think mentioning **`List of Weights and Biases`** makes more sense than mentioning **`List of Variables`**, for better clarity.

**Comment by tusharvickey1999** [comment]
> > @tusharvickey1999,
> > Thank you for your response but still the **documentation** on what exactly **`model.trainable_weights`** or **`layer.trainable_weights`** does and what does that **`function`** **`Return`**, is missing.
> 
> I think mentioning **`List of Weights and Biases`** makes more sense than mentioning **`List of Variables`**, for better clarity.

In a deep neural network, we have weights, biases and several other variable which cannot be classified as weights/biases. therefore we collectively call all those parameter as variable of the network. for eg. batch normalisation has variable which can be learned but it is not weight/bias.
hopefully this answers your query.
